starport/pkg/localfs/search.go

```diff
@@ -1,18 +1,56 @@
 package localfs
 
 import (
+	"os"
+	"path/filepath"
 	"sort"
-
-	"github.com/mattn/go-zglob"
+	"strings"
 )
 
 // Search searches for files in the fs with given glob pattern by ensuring that
 // returned file paths are sorted.
-func Search(pattern string) (paths []string, err error) {
-	files, err := zglob.Glob(pattern)
+func Search(path, pattern string) (paths []string, err error) {
+	files, err := glob(path, pattern)
 	if err != nil {
 		return nil, err
 	}
-	sort.Strings(files)
 	return files, nil
 }
+
+// glob returns the names of all files matching pattern or nil
+// if there is no matching file in all sub-paths recursively. The
+// syntax of patterns is the same as in Match. The pattern may
+// describe hierarchical names such as /usr/*/bin/ed (assuming
+// the Separator is '/').
+func glob(path, pattern string) ([]string, error) {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
```suggestion
	if _, err := os.Stat(path); err != nil && !os.IsNotExist(err) {
```

This should fix the integration tests.